### PR TITLE
fixes and more test of DatabaseMetadata

### PR DESCRIPTION
**Pull request by alexradzin**

### CODE REVIEW

File: src/test/java/com/firebolt/jdbc/metadata/FireboltDatabaseMetadataTest.java [hunk 1]
```diff
@@ -468,6 +700,6 @@ private InputStream getExpectedTypeInfo() {
 	private void getFunctions(CheckedFunction<DatabaseMetaData, String> getter) throws SQLException {
 		String functions = getter.apply(fireboltDatabaseMetadata);
 		assertNotNull(functions);
-		assertTrue(functions.length() > 0);
+        assertFalse(functions.isEmpty());
```
Comment (ptiurin) [inline src/test/java/com/firebolt/jdbc/metadata/FireboltDatabaseMetadataTest.java:703]: nit: something has happened to your indentation here

File: src/integrationTest/java/integration/tests/StatementTest.java [hunk 2]
```diff
@@ -232,4 +237,92 @@ void specialIntValue() throws SQLException {
...
+	 * @throws SQLException if something is going wrong
+	 * see com.firebolt.jdbc.metadata.FireboltDatabaseMetadataTest#nullSorting
+	 */
+	@Test
+	void nullSortOrder() throws SQLException {
+		try (Connection connection = createConnection();
+			 Statement statement = connection.createStatement();
+			 ResultSet rs = statement.executeQuery("select x from (select null as x union all select '' as x union all select 'a' as x) order by x")) {
+			List<String> actuals = new ArrayList<>();
+			while(rs.next()) {
```
Comment (ptiurin) [inline src/integrationTest/java/integration/tests/StatementTest.java:251]: nit: some indent issues here and in the following functions

**Review by ptiurin — APPROVED** [review]
couple of indent problems, otherwise lgtm

File: src/integrationTest/java/integration/tests/StatementTest.java (hunk 2 above)
Comment (alexradzin) [reply src/integrationTest/java/integration/tests/StatementTest.java:251]: I do not see any problem here right now. Let's take a look together?

Comment (ptiurin) [reply src/integrationTest/java/integration/tests/StatementTest.java:251]: image
hope this helps

**Pull request merged**